docs/reference/mapping/types/numeric.asciidoc

```diff
@@ -173,9 +170,20 @@ support this parameter.
     the <<mapping-source-field,`_source`>> field. Accepts `true` or `false`
     (default).
 
-<<mapping-field-meta,`meta`>>::
+include::keyword.asciidoc[tag=dimension]
++
+Of the numeric field types, only `byte`, `short`, `integer`, and `long` fields
+support this parameter.
 
-    Metadata about the field.
+// tag::time_series_metric[]
+`time_series_metric`::
+(Optional, string) For internal use by Elastic only. Marks the field as a time
+series metric. The value is the metric type. Defaults to `null` (Not a time
+series metric).
+// end::time_series_metric[]
++
+For numeric fields, this parameter accepts `gauge` and `counter`. You can't
+update this parameter for existing fields.
```

Comment:
I think we should also mention that for all numeric metrics, `doc_values` parameters must be `true`